spec/requests/api/v1/countries/destroy_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe 'DELETE /api/v1/countries/:id', type: :request do
+  subject { delete api_v1_country_path(country.id), headers: headers, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+
+  before { subject }
+
+  context 'when response is successful' do
```

Comment:
Same about adding another expectation for this context
```ruby
it 'deletes the country' do
  expect { subject }.to change { Country.count }.by(-1)
end

Reply:
Updated